SU2_PY/SU2/eval/design.py

```diff
@@ -226,25 +226,64 @@ def obj_f(dvs,config,state=None):
     
     def_objs = config['OPT_OBJECTIVE']
     objectives = def_objs.keys()
-    
-#    if objectives: print('Evaluate Objectives')
+
     # evaluate each objective
     vals_out = []
     func = 0.0
     for i_obj,this_obj in enumerate(objectives):
         scale = def_objs[this_obj]['SCALE']
         sign  = su2io.get_objectiveSign(this_obj)
-        
         # Evaluate Objective Function
         # scaling and sign
-        func += su2func(this_obj,config,state) * sign * scale
-        
+        if def_objs[this_obj]['CTYPE']=='NONE':
```

Comment:
'CTYPE' is short for 'constraint type' - >, <, =, or 'NONE', where if none it is just the normal objective evaluation - but looking back at this that is not abundantly clear, so I will try to think up a better name for that option.